spring-petclinic-microservices/.mvn/wrapper/MavenWrapperDownloader.java

```diff
@@ -1,118 +0,0 @@
-/*
```

Comment:
Does `petclinic` contains something that spring-boot-3 not supported?

Reply:
Yes, need to refactor the code since the spring-cloud-dependencies 2022.0.0-M4 does not include the spring-cloud-netflix-dependencies